blocks/cards/cards.css

```diff
@@ -134,6 +138,19 @@ main .cards.features-items ul > li {
   align-items: flex-start;
 }
 
+/* Styles for Flat List */
+main .cards.flat-list > ul {
+  flex-direction: row;
+}
+
+main .cards.flat-list .cards-card-image {
+  height: auto;
+}
+
+main .cards.flat-list > ul > li {
+  box-shadow: none;
```

Comment:
You can directly add [this mixin](https://github.com/hlxsites/sunstar/blob/cd9bdb54a6d046d2d6392ef2e3e92d531b2a2fb1/styles/styles.css#L326) to the cards block in word doc
https://github.com/hlxsites/sunstar/blob/cd9bdb54a6d046d2d6392ef2e3e92d531b2a2fb1/styles/styles.css#L326